docs/release-notes/8.7.asciidoc

```diff
@@ -10,6 +10,15 @@
 ==== Known issues
 * After alerts are generated for the first time, you may have to refresh your browser before your alert data appears on pages that use data views (for example, Timeline). Navigating between pages will not work (https://github.com/elastic/security-docs/issues/3046[#3046]).
 
+
+* The {agent} upgrade from 8.6.2 to 8.7.0 might fail for {agents} running the Osquery Manager integration ({agents} can get stuck in the `Updating` state) (https://github.com/elastic/elastic-agent/issues/2433[#2433]). To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after you finish upgrading.
+
++
+If you already upgraded your {agents}, and they're are stuck in the `Updating` state, you will need to upgrade them via API instead of {fleet}. First, remove the Osquery integration from the {agents}, then use the https://petstore.swagger.io/?url=https://raw.githubusercontent.com/elastic/kibana/8.7/x-pack/plugins/fleet/common/openapi/bundled.json#/default/upgrade-agent[Agent upgrade API] to tell the {agents} to upgrade. After the upgrade completes, add the Osquery Manager integration to the {agents} again.
```

Comment:
```suggestion
If you already upgraded your {agents}, and they're are stuck in the `Updating` state, you'll need to upgrade them using the API instead of {fleet}. First, remove the Osquery integration from the {agents}, then use the https://petstore.swagger.io/?url=https://raw.githubusercontent.com/elastic/kibana/8.7/x-pack/plugins/fleet/common/openapi/bundled.json#/default/upgrade-agent[Agent upgrade API] to start the upgrade. After the upgrade completes, add the Osquery Manager integration to the {agents} again.
```